server/src/db_manager.py

```diff
@@ -325,3 +325,57 @@ def create_test_data(self):
             payment1 = Payment("Jim", 45.98, False, 0)
             payment2 = Payment("Tom", 10.00, False, 0)
             self.add_bill("Bob", "Bob", "maccas", 105.98, [payment1, payment2])
+
+    def delete_bill(self, bill_id : int):
+        """
+        Arguments:
+            bill_id {Integer}
+
+        Returns:
+            boolean -- success or fail
+        """
+        conn = sqlite3.connect(self.db_name)
+        c = conn.cursor()
+        try:
+            c.execute("""DELETE FROM bills
+                    WHERE 
+                    bill_id = ?""", [bill_id])
+            conn.commit()
+        except Exception as e:
+            print(e)
+            return False
+        finally:
+            try:
+                conn.close()
+            except UnboundLocalError:
+                pass
+
+    def check_valid_user(self, username : str, bill_id : int):
```

Comment:
I think this method name is a bit ambiguous due to 'valid user'.

We are essentially checking if the user owns the bill here right (and hence, has permission to delete)? How about `check_if_user_owns_bill`? Or something similar :)

Reply:
Yeah, I agree that a more descriptive name would be nice.

Reply:
Cool guys, thanks for the feedback. I've since made the changes.